examples/nodejs/rate-limiter/worker.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  CacheClient,
+  Configurations,
+  CreateCache,
+  CredentialProvider,
+} from "@gomomento/sdk";
+import { IncrementRateLimiter } from "./increment-only-rate-limiter";
+import { DummyService } from "./service";
+import { GetIncrementRateLimiter } from "./get-increment-rate-limiter";
+import { RATE_LIMITER_CACHE_NAME, RateLimiter } from "./rate-limiter";
+import { Metrics } from "./metrics";
+
+async function main() {
+  const momento = await CacheClient.create({
+    configuration: Configurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: "MOMENTO_API_KEY",
+    }),
+    defaultTtlSeconds: 6000,
+  });
+
+  const resp = await momento.createCache(RATE_LIMITER_CACHE_NAME);
+  if (resp instanceof CreateCache.Error) {
+    throw new Error(`Failed to create cache ${RATE_LIMITER_CACHE_NAME}`);
+  }
+
+  // default values
+  let totalRequests = 1000;
+  let randomDelayUpperBound = 60000;
+  let tpmLimit = 5000;
+
+  if (process.argv[2]) {
+    totalRequests = parseInt(process.argv[2], 10);
+  }
+
+  if (process.argv[3]) {
+    randomDelayUpperBound = parseInt(process.argv[3], 10);
+  }
+
+  if (process.argv[4]) {
+    tpmLimit = parseInt(process.argv[4], 10);
+  }
```

Comment:
would it be helpful to include examples of how to pass different CLI args in the readme? might be helpful for users who might want to write a script that tests different rate limiting scenarios

Reply:
yes can add; I actually just added it for my blog demo so not totally applicable but I think I can let this as is and add info about it.